tool_commands/schematisation_checker/command.py

```diff
@@ -85,9 +70,9 @@ def run_it(self, threedi_db, output_file_path):
             ) as output_file:
                 writer = csv.writer(output_file)
                 writer.writerow(
-                    ["id", "table", "column", "value", "description", "type of check"]
+                    ["id", "table", "column", "value", "error_code", "level", "description"]
```

Comment:
could you please change the order to ["level", "error_code", "id", "table", "column", "value", "description"]?
also, 'level' now gives an integer (30 in my test) instead of 'info', 'warning', 'error'